src/test/api/step_definitions/apiSteps.ts

```diff
@@ -0,0 +1,66 @@
+import { Given, When, Then, setDefaultTimeout } from '@cucumber/cucumber';
+import { expect, test, request, APIResponse } from '@playwright/test';
+import { apiFixture } from '../../hooks/apiFixture';
+
+let response: APIResponse;
+const baseURL = process.env.API_BASE_URL
+
+Given('a valid request payload', async function () {
+    // TODO
+});
+
+When('client sends a GET request to {string}', async function (url) {
+    try {
+        response =  await apiFixture.context.get(baseURL + url)    
+    } catch (e) {
+        console.log(e)
+    }
+});
+
+When('client sends a POST request to {string} with body {string}', async function (url, body) {
+    let obj = JSON.parse(body);
+    try {
+        response = await apiFixture.context.post(baseURL + url, {
+            data : obj
+        })  
+    } catch (e) {
+        console.log(e)
+    }
+});
+
+Then('response status should be {string}', async function(responseCode){
```

Comment:
The syntax is not consistent across the project.
```suggestion
Then('response status should be {string}', async function(responseCode) {
```